### ISSUE TITLE
Reset default key bindings

### ISSUE THREAD

**Issue by julietbravo** [OpenEmu/OpenEmu#1995]
An option to restore the default key bindings would be welcome; I just lost mine (SNES, default settings keyboard), can't find them on the OpenEmu website, and can only restore them by guessing and trial and error...

**Comment by julietbravo** [comment]
An overview of the default key bindings somewhere on the website would of course also be sufficient.......

**Comment by loicloic** [comment]
You can restore the default key bindings by deleting this file:

~/Library/Application Support/OpenEmu/Bindings Configurations/Default.oebindings

Note that it will reset all cores key bindings to their defaults.

**Comment by julietbravo** [comment]
I used that solution a couple of days ago, but its a bit of a 'brute force' approach.

**Comment by loicloic** [comment]
I agree but it's a solution nevertheless. I'll label the issue as Feature Request.

**Comment by cristian04** [comment]
+1

**Comment by gbsnkstm** [comment]
Just reboot your computer. It worked for me ..
